Meetings/2022-02-16=meeting-minutes.md

```diff
@@ -0,0 +1,72 @@
+# Maintainers' Meeting 2022-02-16
+
+## Meeting Attendees
+
+- Rob Prouse - @rprouse
+- Mattias Karlsson - @devlead
+- Andrew White - @AndrewTriesToCode
+- Dylan Perks - @Perksey
+- Glenn Watson - @glennawatson
+- James Newton-King - @JamesNK
+- James Turner - @Turnerj
+- Michael Hawker
+- Pascal Berger - @pascalberger
+- Robin Krom
+- Tanner Gooding - @tannergooding
+
+## Recording
+
+**TBD**
+
+### Virtual Inc Staff Observers
+
+- Nadeem Abutaa
+- Tom Pappas
+- Nicole Miller
+
+## Slides
+
+## Agenda
+
+- Review [previous meeting minutes](2022-01-12-meeting-minutes.md)
+- Leaving the .NET Foundation process
+  - Walkthrough the [latest draft](https://github.com/dotnet-foundation/Home/discussions/68#discussioncomment-2168432).
+  - Feedback / Q & A
+  - Road map
+- Signing owner lockout on nuget
+- Centralizing and providing guidance to project maintainers for common questions/issues that arise and how to address them from [this thread here](https://github.com/dotnet-foundation/Home/discussions/61).
+- Other business
+
+## Notes
+
+- Reviewed previous meeting minutes
+- Introduction of new member Robin Krom
+- Discussed the process to leave the foundation,
+  - Tanner Gooding, it would be good to work with the NuGet team to find process for switching to unsigned packages going forward
+  - How do we deal with it when the original maintainers are no longer available?
+  - Can we clarify that it is the non-revokable clause that is preventing us from reassigning the rights of the project and add that to the FAQ?
+  - Forking the project and the foundation not continuing to maintain it is an option to leave the foundation.
+  - Is four weeks notice appropriate?
+  - One comment is that the .NET Foundation has been criticized for not communicating and a project leaving might be considered to be a big change
+  - The four week notice gives the perception that the community has influence in the process
```

Comment:
There was mention that this four week notice could be read only and you didn't need to illicit feedback.